Woof/WoofTests/NetworkLayer/Endpoint/Encodable+AsDictionaryTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+final class EncodableAsDictionaryTests: XCTestCase {
+    func testAsDictionaryMethodSuccessfullyConvertedObjectToDictionary() {
+        struct Model: Encodable {
+            let name: String
+            let age: Int
+        }
+
+        let testModel = Model(name: "John Doe", age: 30)
+
+        do {
+            let dictionary = try testModel.asDictionary()
+
+            XCTAssertEqual(dictionary["name"] as? String, "John Doe")
+            XCTAssertEqual(dictionary["age"] as? Int, 30)
+        } catch {
+            XCTFail("Failed to convert the model to a dictionary: \(error)")
+        }
+    }
```

Comment:
It could be separate cases, but I have added to existing.
```suggestion
  func testAsDictionaryMethodSuccessfullyConvertedObjectToDictionary() throws {
        // Given
        struct Model: Encodable {
            let name: String
            let age: Int
            var fullName: String {
                name + " Ivanov"
            }

            func testFunc() {}
        }
        let testModel = Model(name: "John Doe", age: 30)

        // When
        let dictionary = try testModel.asDictionary()

        // Then
        XCTAssertEqual(dictionary.count, 2)
        XCTAssertEqual(dictionary["name"] as? String, "John Doe")
        XCTAssertEqual(dictionary["age"] as? Int, 30)
    }
```

Reply:
Added to test cases checking the number of records in dictionary.